test_cli.py

```diff
@@ -207,7 +207,17 @@ def _run_command(
 ):
     # We skip using the cli directly if force_subprocess is set to true (which forces
     # us to use subprocess even for cl commands).
-    force_subprocess = not force_subprocess and args[0] == cl
+    if args[0] == cl:
+        force_subprocess = force_subprocess if args[0] == cl else True
+        # By default, request only 4m of memory and 1m of disk so that runs are faster.
```

Comment:
why does this make things faster?

Reply:
Otherwise, the entire run hangs and doesn't actually complete.